extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -219,12 +224,12 @@ public function loadData() {
 						0 => "",
 						1 => $sk->makeLinkObj(
 							Title::newFromText( 'Contributions', NS_SPECIAL ),
-							$wgLang->ucfirst( wfMsg('contribslink') ),
+							$lang->ucfirst( wfMsg('contribslink') ),
 							"target={$oEncUserName}"
 						),
 						2 => $sk->makeLinkObj(
 							Title::newFromText( 'Editcount', NS_SPECIAL ),
-							$wgLang->ucfirst( wfMsg('listusersedits') ),
+							$lang->ucfirst( wfMsg('listusersedits') ),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[project-id]: wikia:app